src/PointwiseFunctions/EquationsOfState/Tabulated.cpp

```diff
@@ -0,0 +1,126 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <array>
+#include <cmath>
+#include <cstddef>
+#include <cstdint>
+#include <hdf5.h>
+#include <memory>
+#include <regex>
+#include <sstream>
+#include <string>
+#include <unordered_map>
+#include <utility>
+#include <vector>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Index.hpp"
+#include "DataStructures/Matrix.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "IO/Connectivity.hpp"
+#include "IO/H5/AccessType.hpp"
+#include "IO/H5/CheckH5.hpp"
+#include "IO/H5/File.hpp"
+#include "IO/H5/Header.hpp"
+#include "IO/H5/Helpers.hpp"
+#include "IO/H5/OpenGroup.hpp"
+#include "IO/H5/ReadTabulated.hpp"
+#include "IO/H5/Version.hpp"
+#include "IO/VolumeDataFile.hpp"
+#include "Informer/InfoFromBuild.hpp"
+#include "PointwiseFunctions/EquationsOfState/Tabulated.hpp"
+#include "Utilities/FileSystem.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+namespace EquationsOfState {
+
+std::vector<double> logrho_from_tabulated() noexcept {
+  h5::H5File<h5::AccessType::ReadOnly> sample_file(
+      unit_test_path() + "PointwiseFunctions/EquationsOfState/KDE_Sample.h5");
```

Comment:
Should this not be taking the `name_` in Tabulated? It looks like the file name is hardcoded in.